tools/scripts/convertScssToJs.js

```diff
@@ -4,9 +4,9 @@
  * Script that processes an scss file containing a set of variables, and
  * generates the equivalent set in JS.
  */
-var fs = require('fs');
-var path = require('path');
-var readline = require('readline');
+var fs = require("fs");
```

Comment:
changes to this file are all fixing prettier errors

Reply:
@Hamms, any insight into why Prettier puts double-quotes here but single quotes in other situations (e.g. `import foo from '...'`?

Reply:
hmm, prettier is actually supposed to prefer single quotes here.

@madelynkasula, is your prettier set up to use our configuration? https://github.com/code-dot-org/code-dot-org/blob/staging/apps/.prettierrc.js

Reply:
yep, it is. this came up because i had a merge conflict, so i merged staging into my branch. when i went to commit the merge, the pre-commit hook brought up all of the syntax errors fixed in this file

when i merged staging into my branch, all i did was `yarn install` -- i didn't do any additional setup for prettier. maybe i missed something?

Reply:
oh, that makes sense. should i leave as-is (with the changes) then?